aws/aws-service-bundler/src/main/java/software/amazon/smithy/java/aws/servicebundle/bundler/AwsServiceBundler.java

```diff
@@ -112,16 +116,17 @@ public Bundle bundle() {
     }
 
     private static String serializeModel(software.amazon.smithy.model.Model model) {
-        return ObjectNode.printJson(ModelSerializer.builder()
+        var output = ObjectNode.printJson(ModelSerializer.builder()
                 .build()
                 .serialize(model));
+        return output;
     }
 
     private static String loadModel(String path) {
         try (var reader = new BufferedReader(new InputStreamReader(
                 Objects.requireNonNull(AwsServiceBundler.class.getResourceAsStream(path)),
                 StandardCharsets.UTF_8))) {
-            return reader.lines().collect(Collectors.joining());
+            return reader.lines().collect(Collectors.joining("\n"));
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
```

Comment:
We should hoist this to Bundler as a protected final method. I've already messed this up twice and I'll keep messing it up every time I rewrite it.